Controller/Adminhtml/Importcategory/Save.php

```diff
@@ -71,147 +79,191 @@ public function __construct(
     }
 
     /**
-     * run the action
+     * @uses given CSV file to import categories
      *
-     * @return \Magento\Backend\Model\View\Result\Redirect
+     * @return Redirect
      */
     public function execute()
     {
 
         $resultRedirect = $this->resultRedirectFactory->create();
         $this->registry->register('isSecureArea', true);
+        
         try {
-            $filepath = $this->_uploadFileAndGetName();
-            if ($filepath!='' && file_exists($filepath)) {
-                chmod($filepath, 0777);
-                $data = $this->_fileCsv->getData($filepath);
+            $filePath = $this->uploadFileAndGetName();
+            
+            if ($filePath!='' && file_exists($filePath)) {
```

Comment:
spacing please

Reply:
and wrap comparasion in `()` for readability please

Reply:
![image](https://user-images.githubusercontent.com/22006850/45302914-701d3f00-b50c-11e8-8edb-0774e9e66149.png)